### PR TITLE
import_cv2

### PR DESCRIPTION
**Pull request by JRMeyer**
change mentions of `import_cv.py` --> `import_cv2.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,13 +253,13 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
 
 The Common Voice corpus consists of voice samples that were donated through Mozilla's [Common Voice](https://voice.mozilla.org/) Initiative.
 
-We provide an importer (`bin/import_cv.py`) which automates downloading and preparing the Common Voice corpus as such:
+We provide an importer (`bin/import_cv2.py`) which automates downloading and preparing the Common Voice (v2.0) corpus as such:
```
Comment (kdavis-mozilla) [inline README.md]: This is not true.

It does not automate downloading. `import_cv.py` automated downloading `import_cv2.py` does not due to the GDPR.

File: README.md [hunk 2]
```diff
@@ -253,13 +253,13 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
...
 
 ```bash
-bin/import_cv.py path/to/target/directory
+bin/import_cv2.py path/to/target/directory
```
Comment (kdavis-mozilla) [inline README.md]: This is not the correct way to call `import_cv2.py`. Looking at the [code](https://github.com/mozilla/DeepSpeech/blob/master/bin/import_cv2.py#L139) it requires tow arguments.

File: README.md [hunk 3]
```diff
@@ -253,13 +253,13 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
...
 ```
 
-If you already downloaded Common Voice from [here](https://voice.mozilla.org/data), simply run `bin/import_cv.py` on the directory where the corpus is located. The importer will detect that you've already downloaded the data and immediately proceed to unpackaging and importing. If you haven't downloaded the data already, `bin/import_cv.py` will download it for you and save to the path you've specified.
+If you already downloaded Common Voice from [here](https://voice.mozilla.org/data), simply run `bin/import_cv2.py` on the directory where the corpus is located. The importer will detect that you've already downloaded the data and immediately proceed to unpackaging and importing. If you haven't downloaded the data already, `bin/import_cv2.py` will download it for you and save to the path you've specified.
```
Comment (kdavis-mozilla) [inline README.md]: > ...simply run `bin/import_cv2.py` on the directory where the corpus is located...

This is not how `import_cv2.py` works. `import_cv2.py` requires two arguments, see [here](https://github.com/mozilla/DeepSpeech/blob/master/bin/import_cv2.py#L139).

Comment (kdavis-mozilla) [inline README.md]: `import_cv2.py` does not automate downloading due to the GDPR.

File: README.md (hunk 1 above)
Comment (JRMeyer) [reply README.md]: 12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8

File: README.md (hunk 3 above)
Comment (JRMeyer) [reply README.md]: 12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8

Comment (JRMeyer) [reply README.md]: 12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8

File: README.md (hunk 2 above)
Comment (JRMeyer) [reply README.md]: 12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8

**Comment by JRMeyer** [comment]
@kdavis-mozilla all your comments are addressed in 12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8

**Pull request merged**